### ISSUE TITLE
feature: Add a section for blogs

### ISSUE THREAD

**Issue by mittal-parth** [mittal-parth/personal-portfolio#39]
It would be nice to have a section that displays the blogs written by the person.
Would like to see some design(s) before coding it out.

**Comment by Anuran12** [comment]
@mittal-parth  I would like to work on this . could you assign this to me ?

**Comment by mittal-parth** [comment]
Yea sure, assigned!

**Comment by Anuran12** [comment]
hey @mittal-parth  when I try to command npm install this error comes 

![image](https://user-images.githubusercontent.com/104062485/194755801-cff75f08-d292-4736-b852-22d120bc77fb.png)

**Comment by mittal-parth** [comment]
@Anuran12 try `npm install --save --legacy-peer-deps`

**Comment by Anuran12** [comment]
@mittal-parth yea problem solved 😁. I have another question...  Is not it better to fetch blogs data from blog website like medium or anything else . then people can write their blogs and paste the api and that's it.

**Comment by Anuran12** [comment]
@mittal-parth So do you have any blog written in somewhere . then send the link

**Comment by mittal-parth** [comment]
> @mittal-parth I have another question... Is not it better to fetch blogs data from blog website like medium or anything else . then people can write their blogs and paste the api and that's it.

hmm yea, that would be great!

**Comment by mittal-parth** [comment]
@Anuran12

> @mittal-parth So do you have any blog written in somewhere . then send the link

No, I dont have blogs written (maybe someday in future 🤞🏼).
The reason behind adding it is so that other people who want to use it as a theme can benefit.
So once this is merged, you wouldn't see changes going live on my portfolio. You can work with any dummy data as of now

**Comment by mittal-parth** [comment]
@mateusabelli that looks really nice! 
Few things:
- I would prefer the image to be slightly bigger
- I think the description can go away then since the image will take bigger space

**Comment by mateusabelli** [comment]
That's cool @mittal-parth I did a few adjustments from your feedback

![image](https://user-images.githubusercontent.com/43862225/195370968-eec93b1e-608a-45fa-9d2c-fa9a279db5e6.png)

**Comment by mittal-parth** [comment]
@mateusabelli awesome
I think you know it but we won't need any of the icons. We can maybe have tags? You can push the code (with or without tags whichever you think looks nice)
